ext/node/polyfills/http.ts

```diff
@@ -655,18 +663,74 @@ class ClientRequest extends OutgoingMessage {
         incoming.url = res.url;
         incoming.statusCode = res.status;
         incoming.statusMessage = res.statusText;
+        incoming.upgrade = null;
+
+        for (const [key, _value] of res.headers) {
+          if (key.toLowerCase() === "upgrade") {
```

Comment:
I guess we don't use primordials in Node code, right?

Reply:
Correct